geopandas/tools/sjoin.py

```diff
@@ -75,6 +76,15 @@ def sjoin(
             " joined".format(index_left, index_right)
         )
 
+    # Check if rtree is installed...
+    package='rtree'
+    if find_spec(package) is None:
+        raise ImportError(
+            f"{package} must be installed to use sjoin\n\n"
+            "See installation instructions at https://geopandas.org/install.html"
```

Comment:
You can reuse the `HAS_RTREE` variable from https://github.com/geopandas/geopandas/blob/master/geopandas/_compat.py here

Reply:
Have swapped em out